tests/test_data.py

```diff
@@ -557,6 +557,16 @@ def test_modeledparameters(db):
     t = db.query(db.ModeledParameters).filter(db.ModeledParameters.c.reference == ref).astropy()
     assert len(t) == 696, f'found {len(t)} modeled parameters with {ref} reference'
 
+    #Test to verify T eff counts
+    ref = 'T eff'
+    t = db.query(db.ModeledParameters).filter(db.ModeledParameters.c.parameter == ref).astropy()
+    assert len(t) == 174, f'found {len(t)} modeled parameters with {ref} parameter'
```

Comment:
```suggestion
    param = 'T eff'
    t = db.query(db.ModeledParameters).filter(db.ModeledParameters.c.parameter == param).astropy()
    assert len(t) == 174, f'found {len(t)} modeled parameters with {param} parameter'
```
let's rename `ref` to be `param` so it's more readable/accurate.